spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "/studios" do 
+  describe "as a visitor" do 
+    it "has all studios name and location" do 
+      uni = Studio.create!(name: "Universal Studios", location: "Hollywood")
+      wb = Studio.create!(name: "Warner Brothers", location: "Burbank")
+      visit "/studios"
+
+      expect(page).to have_content(uni.name)
+      expect(page).to have_content(uni.location)
+      expect(page).to have_content(wb.name)
+      expect(page).to have_content(wb.location)
+
+    end
+  end
+
+    it "shows all the studios movies including title,creation year, and genre" do 
+      uni = Studio.create!(name: "Universal Studios", location: "Hollywood")
+      wb = Studio.create!(name: "Warner Brothers", location: "Burbank")
+      ddarko = uni.movies.create!(title: "Donnie Darko", creation_year: "2000", genre: "Sci-Fi")
+      tdrifter = uni.movies.create!(title: "Tokyo Drifter", creation_year: "1966", genre: "Action")
+      eight = wb.movies.create!(title: "8 1/2", creation_year: "1963", genre: "Fantasy")
+      jodirt = wb.movies.create!(title: "Joe Dirt", creation_year: "2000", genre: "Comedy")
+      visit "/studios"
+
+      
+      expect(page).to have_content(ddarko.title)
```

Comment:
It's nice to use `within` blocks to organize these assertions and make sure this content is in the right place